src/PhoneValidation.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ChangeEvent } from 'react'
+import { useRef, useState } from 'react'
+import BackButton from './components/BackButton'
+import ValidationMessage from './components/ValidationMessage/ValidationMessage'
+// import CorrectValidation from './components/ValidationMessage'
+import './PhoneValidation.scss'
+
+type Props = {}
+
+const PhoneValidation = ({}: Props) => {
+  const inputsRef = [null, null, null, null].map(() => useRef<HTMLInputElement>(null))
+  const [numbers, setNumbers] = useState<number[]>([-1, -1, -1, -1])
+  const buttonRef = useRef<HTMLButtonElement>(null)
+  const [validated, setValidated] = useState<boolean>()
+
+  const OnChangeNumber = (event: ChangeEvent<HTMLInputElement>, index: number) => {
```

Comment:
```suggestion
  const numberChangedHandler = (event: ChangeEvent<HTMLInputElement>, index: number) => {
```

```suggestion
  const onChangeNumber = (event: ChangeEvent<HTMLInputElement>, index: number) => {
```

Personally, I try to name the callback functions of `onSomething` props like in the first suggestion. It's a personal preference so you don't need to abide to it. However, I do recommend to follow the camelCase naming convention for function names